internal/klderrors/errors.go

```diff
@@ -0,0 +1,422 @@
+	// CompilerABISerialize could not serialize the ABI output from solc
+	CompilerABISerialize = "Serializing ABI: %s"
+	// CompilerABIReRead could not re-read serialized output after writing the ABI
+	CompilerABIReRead = "Parsing ABI: %s"
+	// CompilerSerializeDevDocs could not serialize the dev docs output from solc
+	CompilerSerializeDevDocs = "Serializing DevDoc: %s"
+	// ConfigNoRPC missing config for JSON/RPC
+	ConfigNoRPC = "No JSON/RPC URL set for ethereum node"
+	// ConfigKafkaMissingOutputTopic response topic missing
+	ConfigKafkaMissingOutputTopic = "No output topic specified for bridge to send events to"
+	// ConfigKafkaMissingInputTopic request topic missing
+	ConfigKafkaMissingInputTopic = "No input topic specified for bridge to listen to"
+	// ConfigKafkaMissingConsumerGroup consumer group missing
+	ConfigKafkaMissingConsumerGroup = "No consumer group specified"
+	// ConfigKafkaMissingBadSASL problem with SASL config
+	ConfigKafkaMissingBadSASL = "Username and Password must both be provided for SASL"
+	// ConfigKafkaMissingBrokers missing/empty brokers
+	ConfigKafkaMissingBrokers = "No Kafka brokers configured"
+	// ConfigRESTGatewayRequiredReceiptStore need to enable params for REST Gatewya
+	ConfigRESTGatewayRequiredReceiptStore = "MongoDB URL, Database and Collection name must be specified to enable the receipt store"
+	// ConfigRESTGatewayRequiredRPC and RPC stuff
+	ConfigRESTGatewayRequiredRPC = "RPC URL and Storage Path must be supplied to enable the Open API REST Gateway"
+	// ConfigWebhooksDirectRPC for webhooks direct
+	ConfigWebhooksDirectRPC = "No JSON/RPC URL set for ethereum node"
+	// ConfigTLSCertOrKey incomplete TLS config
+	ConfigTLSCertOrKey = "Client private key and certificate must both be provided for mutual auth"
+
+	// ConfigNoYAML missing configuration file on server start
+	ConfigNoYAML = "No YAML configuration filename specified"
+	// ConfigYAMLParseFile failed to parse YAML during server startup
+	ConfigYAMLParseFile = "Unable to parse %s as YAML: %s"
+	// ConfigYAMLPostParseFile failed to process YAML as JSON after parsing
+	ConfigYAMLPostParseFile = "Failed to process YAML config from %s: %s"
+
+	// DeployTransactionMissingCode a DeployTransaction message, without code to deploy
+	DeployTransactionMissingCode = "Missing Compiled Code + ABI, or Solidity"
+
+	// EventStreamsDBLoad failed to init DB
+	EventStreamsDBLoad = "Failed to open DB at %s: %s"
+	// EventStreamsNoID attempt to create an event stream/sub without an ID
+	EventStreamsNoID = "No ID"
+	// EventStreamsWebhookNoURL attempt to create a Webhook event stream without a URL
+	EventStreamsWebhookNoURL = "Must specify webhook.url for action type 'webhook'"
+	// EventStreamsWebhookInvalidURL attempt to create a Webhook event stream with an invalid URL
+	EventStreamsWebhookInvalidURL = "Invalid URL in webhook action"
+	// EventStreamsWebhookInvalidActionType unknown action type
+	EventStreamsWebhookInvalidActionType = "Unknown action type '%s'"
```

Comment:
```suggestion
	EventStreamsWebhookInvalidActionType = "Unknown webhook action type '%s'"
```

Reply:
Going to remove the webhook from the name on this one, as the intention is this is where we'll put `websockets` / `socket.io` support as an alternative to webhooks